django/conf/global_settings.py

```diff
@@ -427,6 +427,8 @@
     'django.middleware.csrf.CsrfViewMiddleware',
 ]
 
+MIDDLEWARES = None
```

Comment:
Yeah, I addressed this in some detail here: https://groups.google.com/d/msg/django-developers/8LMJ44KAxWI/XtnRW2GwEgAJ

tl;dr I think either is defensible, as usage of the term "middleware" is fluid. I prefer `MIDDLEWARE` as the plural, and for now I left it that way in the spec; but I don't feel super-strongly about it.